server/scratchy_server/model/objectId.py

```diff
@@ -0,0 +1,18 @@
+import marshmallow as ma
+import bson
+
+
+class ObjectId(ma.fields.Field):
```

Comment:
Explain why it has been created, that it doesn't work with marshmallow-mongoengine